internal/checkers/helpers_format.go

```diff
@@ -26,15 +26,20 @@ func formatAsCallArgs(pass *analysis.Pass, args ...ast.Expr) []byte {
 	return buf.Bytes()
 }
 
-func formatWithStringCastForBytes(pass *analysis.Pass, e ast.Expr) []byte {
-	if !hasBytesType(pass, e) {
-		return analysisutil.NodeBytes(pass.Fset, e)
+// formatToString formats the expression e as a string.
+func formatToString(pass *analysis.Pass, e ast.Expr) ([]byte, bool) {
```

Comment:
what is difference? what issue is solved?

Reply:
See https://github.com/Antonboom/testifylint/pull/248#discussion_r2336005484